src/components/header/Header.svelte

```diff
@@ -2,9 +2,15 @@
   import WalletConnect from "./WalletConnect.svelte";
   import NetworkSelection from "./NetworkSelection.svelte";
   import { Link, useLocation } from "svelte-navigator";
+  import { oceanUnlockDate } from "../../stores/veOcean";
+  import moment from "moment";
 
-  const aboutURL =
-    "https://blog.oceanprotocol.com/ocean-data-farming-series-c7922f1d0e45";
+  let aboutURL = "https://docs.oceanprotocol.com/veocean-data-farming-tutorial";
+
+  $: if ($oceanUnlockDate && $oceanUnlockDate > moment()) {
```

Comment:
I don't think we need to be this granular with the about link.

Let's just improve the docs and update the about link to point to it.

Reply:
agree, done